test/StakeManager.test.ts

```diff
@@ -0,0 +1,378 @@
+import sourceMapSupport from 'source-map-support'
```

Comment:
any component must import it separately?

Reply:
I will revert this in favor of 0x library fix (see next commit)